File: php/examples/get_filing_methods.php
<?php
require_once 'base_request.php';

// Example of GET /filing-methods

$debug = true;
$company_id   = $_ENV['COMPANY_ID'];
$jurisdiction = $_ENV['JURISDICTION'];
$product_id = $_ENV['FILING_PRODUCT_ID'];
$request_data = '';
$request_params = ['company_id' => $company_id, 'jurisdiction' => $jurisdiction, 'filing_product_id' => $product_id];

send_request('GET', '/filing-methods', $request_params, $request_data);

File: csharp/examples/GetFilingMethods.cs
using System;

namespace Examples.examples
{
    // Example of GET /filing-methods
    
    class GetFilingMethods : BaseRequest
    {
        private string _companyId;
        private string _jurisdiction;
        private string _productId;

        public GetFilingMethods()
        {
            dotenv.net.DotEnv.Load();
            _companyId = Environment.GetEnvironmentVariable("COMPANY_ID");
            _jurisdiction = Environment.GetEnvironmentVariable("JURISDICTION");
            _productId = Environment.GetEnvironmentVariable("FILING_PRODUCT_ID");
            Console.WriteLine($"GetFilingMethods: company_id={_companyId} jurisdiction={_jurisdiction} filing_product_id={_productId}");
        }

        public GetFilingMethods(string companyId, string jursidiction, string productId)
        {
            _companyId = companyId;
            _jurisdiction = jursidiction;
            _productId = productId;
        }

        public override string SendRequest()
        {
            string path = "filing-methods";
            string queryParams = $"?company_id={_companyId}&jurisdiction={_jurisdiction}&filing_product_id={_productId}";
            return GetRequest(path + queryParams);
        }
    }
}
